app/controllers/UserController.scala

```diff
@@ -190,20 +192,23 @@ class UserController @Inject()(val messagesApi: MessagesApi)
     withJsonBodyUsing(userUpdateReader) {
       case (firstName, lastName, email, isActive, isAdmin, assignedMemberships, experiences) =>
         for {
-          user <- UserDAO.findOneById(userId) ?~> Messages("user.notFound")
```

Comment:
Why not keep `findOneById`? I think we use it quite often.

Reply:
Also unpacking with the `Messages("user.notFound")` should be possible in an abstract `findOneById`, maybe having a `match` expression switching the message-identifier.

Reply:
Not sure I understand you here. UserSQLDAO replaces UserDAO here. The findOne methods everywhere expect an ObjectId, which is why there is no explicit “byId” in the name. The “user.notFound” failure message is added here since the SQLDAOs have more specific database-related failure messages

Reply:
:+1: , see #2940